src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -53,12 +54,12 @@ public Line findById(Long id) {
     }
 
     public void update(Line newLine) {
-        String sql = "update LINE set name = ?, color = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getId()});
+        String sql = "update LINE set name = ?, color = ?, extra_fare = ? where id = ?";
+        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getExtraFare(), newLine.getId()});
```

Comment:
```suggestion
        jdbcTemplate.update(sql, newLine.getName(), newLine.getColor(), newLine.getExtraFare(), newLine.getId());
```

이미 아시는 내용일테지만, Object 배열 없이 가변인자를 활용할 수 있답니다 :)